scripts/v.db.renamecolumn/v.db.renamecolumn.html

```diff
@@ -45,8 +45,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 Markus Neteler
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 41

`<a href="v.db.update.html">v.db.update</a><br>`

should be 

`<a href="v.db.update.html">v.db.update</a>,`.

Reply:
Fixed as for `scripts/v.db.addcolumn/v.db.addcolumn.html` (see above)